### ISSUE TITLE
WorldWind is now on GitHub (again)

### ISSUE THREAD

**Issue by wcmatthysen** [unofficial-nasa/world-wind-java-sdk#1]
Hi there,

Don't know if you've seen, but the NASA developers created a new [WorldWind repository](https://github.com/NASAWorldWind/WorldWindJava) on GitHub. The subversion repository has now been abandoned.

**Comment by jdknight** [comment]
Yeah, I really should cleanup/drop these repository clones. I primarily left it to leave an interim source for the RCP-based repositories; however, since all the update processes were based off of SVN, nothing was getting updated anymore. Since it's been almost two (2) years of buffer time, now may be a good time to clean it up.

Thanks for the reminder. I'll schedule some time this month to purge these repositories.
